### PR TITLE
feat: allow to set an `altText` on images

### LINKED ISSUE #624: Add option to add alt-text to images
When a user uploads an image, given them an option to add an alt-text or let them set an alt-text when editing the item.

### PR DESCRIPTION
**Pull request by MalinSvenberg**
closes #624

Requires change to the backend:
- currently schema does not allow to modify the extra of a file or s3File graasp/graasp#422

Screenshot 2023-05-10 at 23 45 51

### CODE REVIEW

**Comment by MalinSvenberg** [comment]
https://github.com/graasp/graasp-builder/assets/56155987/1bce93ac-5e52-47bc-8612-d57a58164636

**Comment by spaenleh** [comment]
@MalinSvenberg Was the change shown in the video removed ? I am not sure if i remember correctly but I think there was an issue with the radio button not making it clear that it was simply to toggle between the description and the alt text. Correct me if I am wrong.

**Comment by MalinSvenberg** [comment]
@spaenleh Yes the change in the video was removed. I went back to the first version instead, with only the Name and the Alt text input.

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "@emotion/styled": "11.11.0",
     "@graasp/chatbox": "2.0.0-rc.1",
     "@graasp/query-client": "1.0.0",
-    "@graasp/sdk": "1.0.0",
+    "@graasp/sdk": "github:graasp/graasp-sdk#162-add-alttext-to-settings-fileitem",
```
Comment (pyphilia) [inline package.json:17]: Update.

File: package.json [hunk 2]
```diff
@@ -140,6 +140,7 @@
     "webpack-bundle-analyzer": "4.9.0"
   },
   "resolutions": {
+    "@graasp/sdk": "github:graasp/graasp-sdk#162-add-alttext-to-settings-fileitem",
```
Comment (pyphilia) [inline package.json:143]: Update.

File: src/components/context/EditItemModalContext.tsx [hunk 3]
```diff
@@ -100,6 +114,57 @@ const EditItemModalProvider = ({ children }: Props): JSX.Element => {
...
+        <BaseItemForm
+          onChange={setUpdatedItem}
+          item={fileItem}
+          updatedProperties={updatedProperties}
+        />
+        {mimetype && MimeTypes.isImage(mimetype) && (
+          <TextField
+            variant="standard"
+            id={ITEM_FORM_IMAGE_ALT_TEXT_EDIT_FIELD_ID}
+            label="Alternative Text (for accessibility purposes)"
```
Comment (pyphilia) [inline src/components/context/EditItemModalContext.tsx:144]: Translate.

**Review by pyphilia — APPROVED** [review]
LGTM 🌺

**Pull request merged**